lib/scraper/bedelias.rb

```diff
@@ -9,8 +9,9 @@ module Bedelias
     include Capybara::DSL
 
     # 5265 - CIENCIAS HUMANAS Y SOCIALES - min: 10 créditos
+    GROUP_CODE_NAME_CREDITS_REGEX = /\A(\w+) - (.+) - (?:min): (\d+)( créditos)?\z/
     # SRN14 - MATEMÁTICA DISCRETA I - créditos: 10
-    CODE_NAME_CREDITS_REGEX = /\A(\w+) - (.+) - (?:min|créditos): (\d+)( créditos)?\z/
+    SUBJECT_CODE_NAME_CREDITS_REGEX = /\A(\w+) - (.+) - (?:créditos): (\d+)(?: programa)?\z/
```

Comment:
just to confirm, the `?:` in `(?: programa)?` is just to not make it part of the `captures` ?

Reply:
Yeah exactly, we don't need to capture it in a group.

Reply:
can we do the same for the group regex to keep it consistent? or remove from both?

Reply:
Yeah totally. Updated in c8bb7c883d3effd6d12a533e3d9face263ea894b